### ISSUE TITLE
Out-of-date Documentation for Editable Installs

### ISSUE THREAD

**Issue by BrutalSimplicity** [pdm-project/pdm#1357]
[Here](https://pdm.fming.dev/latest/usage/dependency/#editable-dependencies), the documentation still mentions that editable installs can be installed using:

```shell
# A relative path to the directory
pdm add -e ./sub-package
# A file URL to a local directory
pdm add -e file:///path/to/sub-package
# A VCS URL
pdm add -e git+https://github.com/pallets/click.git@main#egg=click
```
This seems to be incorrect following the [v2 update](https://github.com/pdm-project/pdm/blob/main/CHANGELOG.md#breaking-changes-1)

**Comment by frostming** [comment]
Right, it would make sense if we add the `--dev` or flag, would you like to contribute?

**Comment by BrutalSimplicity** [comment]
I've added a PR for this update. Thanks for the suggestion.

**Comment by frostming** [comment]
Closed by #1358
